### PR TITLE
Switch to forked versions of zookeeper and kafka

### PR DESCRIPTION
**Pull request by Zapadlo**
- New version of kafka (0.9.0.1) direct fork of
   wurstmeister/kafka-docker

 - Same version of zookeeper as current prod (3.4.6) with additional
   Dockerfile changes to reduce layers/image size

https://github.com/Financial-Times/kafka-docker
https://github.com/Financial-Times/zookeeper-docker

### CODE REVIEW

File: kafka.service [hunk 1]
```diff
@@ -8,11 +8,11 @@ TimeoutStartSec=0
...
-ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p > /dev/null 2>&1'
-ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p > /dev/null 2>&1'
-ExecStartPre=/usr/bin/docker pull wurstmeister/kafka:0.8.2.0
-ExecStart=/bin/bash -c "\
-  /usr/bin/docker run --rm --name %p -p 9092:9092 --link zookeeper:zk --env \"KAFKA_ADVERTISED_HOST_NAME=$$HOSTNAME\" --env=\"KAFKA_MESSAGE_MAX_BYTES=16777216\" --env=\"KAFKA_REPLICA_FETCH_MAX_BYTES=16777216\" -v /vol/kafka:/kafka -v /var/run/docker.sock:/var/run/docker.sock wurstmeister/kafka:0.8.2.0"
+ExecStartPre=-/bin/sh -c '/usr/bin/docker kill %p >/dev/null 2>&1'
+ExecStartPre=-/bin/sh -c '/usr/bin/docker rm %p >/dev/null 2>&1'
+ExecStartPre=/bin/sh -c 'docker history coco/kafka-docker:0.9.0.1 >/dev/null 2>&1 || docker pull coco/kafka-docker:0.9.0.1'
+ExecStart=/bin/sh -c '\
+  docker run --rm --name %p -p 9092:9092 --link zookeeper:zk -e "KAFKA_ADVERTISED_HOST_NAME=$$HOSTNAME" -e="KAFKA_MESSAGE_MAX_BYTES=16777216" -e="KAFKA_REPLICA_FETCH_MAX_BYTES=16777216" -v /vol/kafka:/kafka -v /var/run/docker.sock:/var/run/docker.sock coco/kafka-docker:0.9.0.1'
```
Comment (khatton-ft) [inline kafka.service]: `-e` ... or `-e=`... ?

**Comment by kovacshuni** [comment]
+1

File: kafka.service (hunk 1 above)
Comment (Zapadlo) [inline kafka.service]: Either seems to work:
```
~/docker run -e "TEST=123" -ti alpine sh -c 'echo $TEST'
123
~/docker run -e="TEST=123" -ti alpine sh -c 'echo $TEST'
123
~/
```
But doesn't make sense to have them inconsistent.

**Comment by khatton-ft** [comment]
:+1:

**Comment by Zapadlo** [comment]
Deployed to `xp`

**Comment by Nesze** [comment]
Reverted change in `xp`. Publishing didn't work, because the cms-notifier couldn't put the message on the queue. Didn't inspect it further, it might be only a config issue?

**Comment by Zapadlo** [comment]
Its okay, thanks, I will deploy to an isolated environment and test there. Was hoping for a seemless upgrade.

**Pull request closed without merging**